examples/chart/teleport-demo/docker/build-all.sh

```diff
@@ -1,15 +1,25 @@
-#!/usr/bin/env bash
+#!/bin/bash
 set -e
+ONE=$1
+TWO=$2
 VERSION=3.2.0
 if [[ "$1" != "" ]]; then
     VERSION=$1
     shift
 fi
-set -e
+GCPROJECT=kubeadm-167321
+if [[ "" != "${TWO}" ]]; then
+    GCPROJECT=${TWO}
+    shift
+fi
+echo "${VERSION} version"
+echo "${GCPROJECT} project"
+
```

Comment:
Here's a better way to be able to provide multiple options via the command line.

```bash
VERSION=4.2.3
GCPROJECT=kubeadm-167321

usage() { echo "Usage: $(basename $0) <-v [version]> <-p [project]> 1>&2; exit 1; }
while getopts ":v:p:" opt; do
    case "${opt}" in
        v)
            VERSION=${OPTARG}
            ;;
        p)
            GCPROJECT=${OPTARG}
            ;;
        *)
            usage
            ;;
    esac
done
shift $((OPTIND-1))
```

This will enable you to call the script in any of these ways and have them work as expected without assuming an order of arguments:

```bash
./build-all.sh
./build-all.sh -v 4.2.3
./build-all.sh -p newproject-123456
./build-all.sh -v 4.2.3 -p newproject-123456
```